packages/wonder-blocks-button/src/themes/default.ts

```diff
@@ -339,12 +339,13 @@ const theme = {
     font: {
         size: {
             // NOTE: This token is specific to this button size.
-            large: 18,
+            large: "1.8rem",
         },
         lineHeight: {
-            small: tokens.font.lineHeight.small + textUnderlineOffset,
-            default: tokens.font.lineHeight.medium + textUnderlineOffset,
-            large: tokens.font.lineHeight.medium + 2 + textUnderlineOffset,
+            small: tokens.font.lineHeight.xMedium,
+            default: tokens.font.lineHeight.large,
```

Comment:
`default` button line-height is now `sizing.size_240`.  (2.4rem / 24px)

Previously it used `tokens.font.lineHeight.medium` (`20px`) + `textUnderlineOffset` (`4`) = `24px` so it should be equivalent.